app/classifier/index.cjsx

```diff
@@ -64,14 +67,38 @@ Classifier = React.createClass
       @context.geordi?.forget ['subjectID']
 
   loadSubject: (subject) ->
-    @setState subjectLoading: true
+    @setState
+      expertClassification: null
+      selectedExpertAnnotation: -1
+      showingExpertClassification: false
+      subjectLoading: true
+
+    if @props.project.experimental_tools.indexOf('expert comparison summary') > -1
+      @getExpertClassification @props.workflow, @props.subject
 
     preloadSubject subject
       .then =>
         if @props.subject is subject # The subject could have changed while we were loading.
           @setState subjectLoading: false
           @props.onLoad?()
 
+  getExpertClassification: (workflow, subject) ->
+    awaitExpertClassification = Promise.resolve do =>
+      apiClient.get('/classifications/gold_standard', {
+        workflow_id: workflow.id,
+        subject_ids: [subject.id]
+      })
+        .catch ->
+          []
+        .then ([expertClassification]) ->
+          expertClassification
+
+    awaitExpertClassification.then (expertClassification) =>
+      expertClassification ?= subject.expert_classification_data?[workflow.id]
```

Comment:
this looks good, just more a question for my general understanding - we're trying to get the expert classification from the API above (line 87), but if the response is null or undefined, we try to get it from the subject itself, specifically an `expert_classification_data` property?